Scripts/assignment/datatypes/freight_specification.py

```diff
@@ -69,7 +69,7 @@ def __init__(self,
                 "perception_factor": 1,
             },
             "aux_transit_time": {
-                "perception_factor": 10,
+                "perception_factor": 30,
```

Comment:
This is related to having difficulties interpreting aux time matrix results in current iteration due to timau * 3 in mode u. Instead I tested demand calc with having timau * 1 and perception factor 30 here which seemed to produce same results with easier aux time interpretation. We can drop this from this development section if needed.